ddtrace/llmobs/_llmobs.py

```diff
@@ -152,14 +152,14 @@ def enable(
         :param str env: Your environment name.
         :param str service: Your service name.
         """
-        if cls.enabled:
-            log.debug("%s already enabled", cls.__name__)
-            return
-
         if os.getenv("DD_LLMOBS_ENABLED") and not asbool(os.getenv("DD_LLMOBS_ENABLED")):
             log.debug("LLMObs.enable() called when DD_LLMOBS_ENABLED is set to false or 0, not starting LLMObs service")
+            cls.enabled = False
```

Comment:
I am not sure whether this change will fix my issue, but I think the class should always be disabled if DD_LLMOBS_ENABLED is set to False.
https://github.com/DataDog/dd-trace-py/issues/10859